.github/CODEOWNERS

```diff
@@ -359,6 +359,7 @@
 
 # Rich text editor team.
 /core/templates/dev/head/components/ck-editor-helpers/ck-editor-4-rte.directive.ts @aks681
+/core/templates/dev/head/components/ck-editor-helpers/ck-editor-5-rte.directive.ts @apb7 @NishealJ
```

Comment:
Hmm, I don't think I should be the code owner for this file. It would be better if you replace me with @aks681. (@aks681, does that sound fine?)

Reply:
Hi @aks681, is that fine?

Reply:
Considering he owns ck-editor-4-rte, I think yes.

Reply:
Done Added @aks681 Thanks!